eplmp-server/eplmp-server-rest/src/main/java/org/polarsys/eplmp/server/rest/writers/QueryResultMessageBodyWriter.java

```diff
@@ -228,9 +230,13 @@ private void generateJSONResponse(OutputStream outputStream, QueryResult queryRe
                 }
 
                 jg.write(QueryField.PART_ITERATION_LINKED_DOCUMENTS, sb.toString());
+                // Clearing the StringBuilder
+                sb.delete(0, sb.length());
```

Comment:
Why such operation?
This StringBuilder is local to the if bloc, no need to clear it.